src/Core/Domain/Product/Stock/Exception/CannotAddStockAvailableException.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\Stock\Exception;
+
+class CannotAddStockAvailableException extends ProductStockException
```

Comment:
i know php doc would be redundant here, but there are some guys that always asks them :smile:

Reply:
> but there are some guys that always asks them 
👀 

> I wonder if we really need to add them everywhere
Only if they bring relevant information (for example if name is ambiguous, it can help clearing the confusion). The phpDoc must add documentation value, else it can go away.

Reply:
I added every missing PHPDoc ^^ Even if the name seem clear an additional detail can't hurt
And mostly I bother you so much @zuk3975 with those PHPDocs than I can't look the other way when I'm reminded this 😅